packages/pilot/src/components/Operations/index.js

```diff
@@ -135,11 +135,12 @@ class Operations extends PureComponent {
       <div className={style.container}>
         <div className={style.head}>
           <CardTitle
+            className={style.cardContent}
             subtitle={this.renderSubTitle()}
             title={this.renderTitle()}
           />
         </div>
-        <CardContent>
+        <CardContent className={style.cardContent}>
```

Comment:
Este card content é desnecessário, ele pode ser removido dado que sejam adicionados alguns estilos para suprir isso, segue a sujestão que deve ser aplicada no [arquivo de estilos](packages/pilot/src/components/Operations/style.css) na linha 14.
```suggestion
       .container {
  border-radius: 3px;

  & > div:last-child {
    padding-top: var(--spacing-medium);
  }

  & .head > div:first-child {
    padding-left: 0;
    padding-right: 0;
    padding-top: 12px;
  }
}
```

Reply:
Boa! Apliquei a alteração e removi a classe que deixou de ser utilizada